packages/chai-openapi-response-validator/test/assertions/satisfyApiSpec/openapi2SpecsDefineBasePath.test.js

```diff
@@ -0,0 +1,97 @@
+/** *****************************************************************************
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ****************************************************************************** */
+
+const chai = require('chai');
+const path = require('path');
+
+const chaiResponseValidator = require('../../..');
+
+const { expect, AssertionError } = chai;
+
+describe('Using OpenAPI 2 specs that define a basePath', () => {
+  before(() => {
+    chai.use(chaiResponseValidator(path.resolve(
+      '../../commonTestResources/exampleOpenApiFiles/valid/'
+         + 'openapi2DefineBasePath.yml'
```

Comment:
```suggestion
         , 'openapi2DefineBasePath.yml'
```
(and then put the comma on the preceding line)

Reply:
This was resolved by the ["Refactored basePath test " commit](https://github.com/RuntimeTools/OpenAPIValidators/pull/160/commits/126d6dd0ba46e327f8ec1bd35a2b837b319d98f6).